cmd/cli/serve/serve.go

```diff
@@ -250,6 +258,17 @@ func serve(cmd *cobra.Command) error {
 		nodeConfig.RequesterTLSKeyFile = key
 	}
 
+	// Start up Dashboard
+	if startWebUI {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		err := StartWebUIServer(ctx)
```

Comment:
This method is blocking since it waits for the command to complete or the context to be canceled (same is true for the simplified version of the method I mentioned below). We'll need to either call this method via a go routine, or make it non-blocking else the node will never start when the webUI is enabled. We can't merge this until this issue is addressed.

Typically methods prefixed with `Start*` imply they are non-blocking - they start something and then return, conversely methods prefixed with `Run*` are usually blocking. The method name should reflect its behavior.